### PR TITLE
update kinds

### PR DESCRIPTION
**Pull request by csantanapr**

### CODE REVIEW

File: core/controller/src/main/resources/apiv1swagger.json [hunk 1]
```diff
@@ -1554,13 +1554,10 @@
...
-                        "nodejs",
                         "nodejs:6",
-                        "nodejs:default",
-                        "python",
-                        "swift",
-                        "swift:3",
-                        "swift:default",
+                        "python:2",
+                        "python:3",
+                        "swift:3.1.1",
```
Comment (rabbah) [inline core/controller/src/main/resources/apiv1swagger.json]: dropped swift 3.

Comment (csantanapr) [reply core/controller/src/main/resources/apiv1swagger.json]: it's encouragement documentation for folks to use `swift:3.1.1` :-)
I can add back, and remove later when we deprecate and announce in mailing list

**Comment by csantanapr** [comment]
@rabbah added `swift:3` please review

**Comment by csantanapr** [comment]
PG 3 1342

**Pull request merged**